src/test/groovy/org/prebid/server/functional/util/AllureReporter.groovy

```diff
@@ -253,4 +272,46 @@ class AllureReporter extends AbstractRunListener implements IGlobalExtension {
         lifecycle.stopTestCase(uuid)
         lifecycle.writeTestCase(uuid)
     }
+
+
+    private void writeBlocks(FeatureInfo feature, IterationInfo iteration) {
+        def blockText = StringUtils.EMPTY
+        for (BlockInfo block : feature.blocks) {
+            if (!isEmptyOrContainsOnlyEmptyStrings(block.texts)) {
+                def size = block.texts.size()
+                for (int i = 0; i < size; i++) {
+                    if (iteration != null) {
+                        blockText = generationParams(block.texts[i], feature.dataVariables, iteration)
```

Comment:
you may declare blockText variable here and initialize its value (EMPTY or generationParams()) using a ternary operator